### DISCUSSION TITLE
Wallos Discussions!

### DISCUSSION THREAD

**Discussion by ellite** [ellite/Wallos#200] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chanraerae** [comment]
is there any way to have AED in currency?

**Reply by ellite** [reply]
Hi. Isn't it already there?
If not, you can go to settings and add a new currency. Use the code AED.

**Reply by therealwave** [reply]
I just want to say this program is awesome! Simple and strait forward. Thanks and good job.
